taxi/admin.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from taxi.models import Manufacturer, Driver, Car
+
+
+class DriverAdmin(admin.ModelAdmin):
+    list_display = UserAdmin.list_display + ("license_number",)
+
+    fieldsets = (

```

Comment:
Please use the `fieldsets `and `add_fieldsets `attributes from `UserAdmin`, as you have already done with list_display.